hack/test-stirunimage.sh

```diff
@@ -14,6 +14,10 @@ function time_now()
 
 mkdir -p /tmp/sti
 WORK_DIR=$(mktemp -d /tmp/sti/test-work.XXXX)
+S2I_WORK_DIR=${WORK_DIR}
+if [[ "$OSTYPE" == "cygwin" ]]; then
```

Comment:
I'm fine with that. Since this is good news for anyone on Windows, I'd suggest adding some documentation in the README or some other document. And a note in the next release /cc @bparees :smile: